layouts/index.html

```diff
@@ -0,0 +1,21 @@
+{{ define "main" }}
+  <h2>Welcome to the experimental space of Water Data for the Nation!</h2>
+  <p>This is an area where the USGS Internet of Water software development team can try out new things, and users can see what we are working on before it is launched on one of the production sites.  Most of the services, tools and visualiztions that you see here will eventually make to the core waterdata.usgs.gov or waterservices.usgs.gov sites, but some may not. </p>
```

Comment:
This part of the sentence 'eventually make to the core waterdata.usgs.gov' should probably read 'eventually make IT to the core waterdata.usgs.gov ' (the same thing is true for the text on the main page).